pkg/key/key.go

```diff
@@ -3,9 +3,9 @@ package key
 // These consts are used by template.Params.Validate() to check that provided versions
 // fall within the known-good ranges.
 const (
-	CalicoVersionConstraint                       = ">= 3.10.0 < 3.16.0"
+	CalicoVersionConstraint                       = ">= 3.16.0 < 3.17.0"
```

Comment:
Change to CNI installation command is not backwards compatible so 3.16 is the new minimum and this will be a major release.